x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/configure_pipeline.tsx

```diff
@@ -79,21 +76,6 @@ export const ConfigurePipeline: React.FC = () => {
 
   const nameError = formErrors.pipelineName !== undefined && pipelineName.length > 0;
 
-  const modelOptions: Array<EuiSuperSelectOption<string>> = [
-    {
-      disabled: true,
-      inputDisplay:
-        existingPipeline && pipelineName.length > 0
-          ? MODEL_REDACTED_VALUE
-          : MODEL_SELECT_PLACEHOLDER,
-      value: MODEL_SELECT_PLACEHOLDER_VALUE,
-    },
```

Comment:
This logic can be safely removed:
- The "Select model" placeholder is no longer relevant in the selection list component
- The new/existing pipeline choice will be split into separate tabs in a follow-up PR. Therefore if an existing pipeline is selected with a redacted model, the model selection component won't be rendered in that tab, and there's no need for supporting the disabled placeholder.